package.json

```diff
@@ -25,9 +25,9 @@
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-plugin-webpack-alias": "^2.1.2",
     "babel-preset-react": "^6.24.1",
-    "bitcore-lib": "^0.15.0",
-    "bitcore-mnemonic": "^1.5.0",
+    "bip39": "^2.5.0",
     "es6-promise": "^4.2.4",
+    "ethereumjs-wallet": "^0.6.0",
```

Comment:
I'm not in principel against using this, but we are trying to econmize on package size, and we can perhaps use https://www.npmjs.com/package/hdkey (I  mention that because ethereumjs-wallet uses that https://github.com/ethereumjs/ethereumjs-wallet/blob/master/package.json#L37 and it seems we are only using that part)